src/api/transact-namespace.ts

```diff
@@ -91,6 +101,52 @@ export class TransactNamespace {
     );
   }
 
+  /**
+   * Used to simulate a single transaction.
+   *
+   * Returns list of asset changes.
+   *
+   * @param transaction The transaction to simulate.
+   */
```

Comment:
pls include jsdoc for `blockIdentifier` here and below

Reply:
yup yup - aware that's missing :)